_conferences/womenwhocodemobilesummit.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Women Who Code Mobile Summit 2022"
+website: https://www.womenwhocode.com/mobile/about
```

Comment:
This should be the website for the event. I suggest using the other URL as that is what allows people to register for the event.

```suggestion
website: https://hopin.com/events/women-who-code-mobile-summit/registration
```